docs/EN/Configuration/OmnipodDASH.md

```diff
@@ -1,91 +1,567 @@
+
+
+|Deactivate_Pod_5|
+
+4. Click on the green button to return to the **Pod Management** screen.
+
+
+|Deactivate_Pod_6|
+
+5. You are now returned to the **Pod Management** menu; press the back button on your phone to return to the **DASH** tab. Verify that the **Pod status:** field displays a **No active Pod** message.
+
+
+|Deactivate_Pod_7||Deactivate_Pod_8|
+
+
+## Suspending and Resuming Insulin Delivery
+
+The process below will show you how to suspend and resume insulin pump delivery.
+
+*NOTE - if you do not see a SUSPEND button*, then it has not been enabled to display in the DASH tab. Enable the **Show Suspend Delivery button in DASH tab** setting in the [Dash settings](#Dash-settings)under **Other**.
+
+## Suspending Insulin Delivery
+
+Use this command to put the active pod into a suspended state. In this suspended state, the pod will no longer deliver any insulin. This command mimics the suspend function that the original Omnipod Dash PDM issues to an active pod.
+
+1. Go to the **DASH (POD)** tab and click on the **SUSPEND (1)** button. The **SUSPEND (3)** button will become greyed out, and the **Pod Status (2)** will display **SUSPEND DELIVERY**.
+
+
+|Suspend_1||Suspend_2|
+
+2. When the suspend command is successful, a confirmation dialog will display the message **All insulin delivery has been suspended**. Click **OK** to confirm and proceed.
+
+|Suspend_3|
+
+3. Your active pod has now suspended all its insulin delivery. The **DASH** tab will update the **Pod Status (1)** to **Suspended**. The **SUSPEND** button will change to a new **RESUME DELIVERY (2)** button.
+
+## Resuming Insulin Delivery
+
+Use this command to instruct the active, currently suspended pod to resume insulin delivery. After the command is successfully processed, insulin will resume normal delivery using the current basal rate based on the current time from the active basal profile. The pod will again accept commands for bolus, TBR, and SMB.
+
+1. Go to the **DASH** tab and ensure the **Pod status (1)** field displays **SUSPENDED**, then press the **RESUME DELIVERY (2)** button to start the process to instruct the current pod to resume normal insulin delivery. A message **RESUME DELIVERY** will display in the **Pod Status (3)** field.
+
+
+|Resume_1||Resume_2|
+
+2. When the Resume delivery command is successfully confirmed a confirmation dialog will display the message **Insulin delivery has been resumed**. Click **OK** to confirm and proceed.
+
+
+|Resume_3|
+
+3. The **DASH** tab will update the **Pod status (1)** field to display **RUNNING,** and the **Resume Delivery** button will now display the **SUSPEND (2)** button.
+
+
+|Resume_4|
+
+## Acknowledging Pod Alerts
+
+*NOTE - The Silence ALERTS button is only available on the **DASH** tab when the pod expiration or low reservoir alert has been triggered.*
```

Comment:
If the `Silence ALERTS` button is not visible and you hear beep sounds from the pod, try to 'Refresh pod status'.